packages/remix-vercel/server.ts

```diff
@@ -1,4 +1,7 @@
-import type { VercelRequest, VercelResponse } from "@vercel/node";
+import type {
+  IncomingMessage as VercelRequest,
+  ServerResponse as VercelResponse,
+} from "node:http";
```

Comment:
This would be a breaking change, because TS will throw an error if people are implementing `getLoadContext` and using the added properties

```ts
export declare type VercelRequest = IncomingMessage & {
    query: VercelRequestQuery;
    cookies: VercelRequestCookies;
    body: VercelRequestBody;
};
export declare type VercelResponse = ServerResponse & {
    send: (body: any) => VercelResponse;
    json: (jsonBody: any) => VercelResponse;
    status: (statusCode: number) => VercelResponse;
    redirect: (statusOrUrl: string | number, url?: string) => VercelResponse;
};
```

Reply:
Now vercel users cannot upgrade remix until `@vercel/remix` package is bumped as well because it has fixed `@remix-run/node` dependency. Would be easier for everyone if remix just integrated all the stuff back from vercel fork.